stackoverflow-dark.user.css

```diff
@@ -44,7 +44,7 @@ crosshatch "Cross-Hatch" <<<EOT
 }
 @advanced dropdown syntax-theme "Syntax theme" {
   Desert "Desert" <<<EOT
-  /*! Desert *\/pre.prettyprint{display:block;background-color:#333!important}pre .nocode{background-color:none!important;color:#000!important}pre .str,code .str{color:#ffa0a0!important}pre .kwd,code .kwd{color:#f0e68c!important;font-weight:bold!important}pre .com,code .com{color:#87ceeb!important}pre .typ,code .typ{color:#98fb98!important}pre .lit,code .lit{color:#cd5c5c!important}pre .pun,code .pun{color:#fff!important}pre .pln,code .pln{color:#fff!important}pre .tag,code .tag{color:#f0e68c!important;font-weight:bold!important}pre .atn,code .atn{color:#bdb76b!important;font-weight:bold!important}pre .atv,code .atv{color:#ffa0a0!important}pre .dec,code .dec{color:#98fb98!important}ol.linenums{margin-top:0;margin-bottom:0;color:#aeaeae}li.L0,li.L1,li.L2,li.L3,li.L5,li.L6,li.L7,li.L8{list-style-type:none!important}@media print{pre.prettyprint{background-color:none!important}pre .str,code .str{color:#060!important}pre .kwd,code .kwd{color:#006!important;font-weight:bold!important}pre .com,code .com{color:#600!important;font-style:italic!important}pre .typ,code .typ{color:#404!important;font-weight:bold!important}pre .lit,code .lit{color:#044!important}pre .pun,code .pun{color:#440!important}pre .pln,code .pln{color:#000!important}pre .tag,code .tag{color:#006!important;font-weight:bold!important}pre .atn,code .atn{color:#404!important}pre .atv,code .atv{color:#060!important}} EOT;
+  /*! Desert *\/pre.prettyprint{display:block;background-color:#333!important}pre .nocode{background-color:transparent!important;color:#000!important}pre .str,code .str{color:#ffa0a0!important}pre .kwd,code .kwd{color:#f0e68c!important;font-weight:bold!important}pre .com,code .com{color:#87ceeb!important}pre .typ,code .typ{color:#98fb98!important}pre .lit,code .lit{color:#cd5c5c!important}pre .pun,code .pun{color:#fff!important}pre .pln,code .pln{color:#fff!important}pre .tag,code .tag{color:#f0e68c!important;font-weight:bold!important}pre .atn,code .atn{color:#bdb76b!important;font-weight:bold!important}pre .atv,code .atv{color:#ffa0a0!important}pre .dec,code .dec{color:#98fb98!important}ol.linenums{margin-top:0;margin-bottom:0;color:#aeaeae}li.L0,li.L1,li.L2,li.L3,li.L5,li.L6,li.L7,li.L8{list-style-type:none!important}@media print{pre.prettyprint{background-color:transparent!important}pre .str,code .str{color:#060!important}pre .kwd,code .kwd{color:#006!important;font-weight:bold!important}pre .com,code .com{color:#600!important;font-style:italic!important}pre .typ,code .typ{color:#404!important;font-weight:bold!important}pre .lit,code .lit{color:#044!important}pre .pun,code .pun{color:#440!important}pre .pln,code .pln{color:#000!important}pre .tag,code .tag{color:#006!important;font-weight:bold!important}pre .atn,code .atn{color:#404!important}pre .atv,code .atv{color:#060!important}} EOT;
```

Comment:
You should not edit the themes in  `stackoverflow-dark.user.css` you edit https://github.com/StylishThemes/StackOverflow-Dark/tree/master/themes desired theme, else, the changes are lost.